students/Chao/session03/list_lab.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+
+# Creating global fruit list
+fruits = ["Apples", "Pears", "Oranges", "Peaches"]
+
+def series_1():
+    """ Series 1 tasks """
+    
+    print("The current fruit list: " + str(fruits))
+    
+    # Append a fruit
+    morefruit = input("Enter the fruit you want to add to the list: ")
+    fruits.append(morefruit)
+    print("The current fruit list: " + str(fruits) + '\n')
+    
```

Comment:
Cool, did not realize you could do this. Alternatively, if you want the list of fruits to be more sentence-like you could do:
```
print("The current fruit list: " + ", ".join(fruits) + '\n')
```

Reply:
Or you can also do `print("The current fruit list: ", fruits, "\n")`